Resources/Prototypes/Objectives/traitor.yml

```diff
@@ -280,16 +280,17 @@
     stealGroup: WeaponAntiqueLaser
     owner: job-name-captain
 
-- type: entity
-  parent: BaseCaptainObjective
-  id: NukeDiskStealObjective
-  components:
-  - type: Objective
-    # high difficulty since the hardest item both to steal, and to not get caught down the road,
-    # since anyone with a pinpointer can track you down and kill you
-    # it's close to being a stealth loneop
-    difficulty: 4
-  - type: NotCommandRequirement
-  - type: StealCondition
-    stealGroup: NukeDisk
-    owner: objective-condition-steal-station
+# - type: entity
+  # noSpawn: true
+  # parent: BaseCaptainObjective
+  # id: NukeDiskStealObjective
+  # components:
+  # - type: Objective
+    # # high difficulty since the hardest item both to steal, and to not get caught down the road,
+    # # since anyone with a pinpointer can track you down and kill you
+    # # it's close to being a stealth loneop
+    # difficulty: 4
+  # - type: NotCommandRequirement
+  # - type: StealCondition
+    # stealGroup: NukeDisk
+    # owner: objective-condition-steal-station
```

Comment:
```suggestion
# Corvax-MRP-Start: Make nuke disk objective separate
# - type: entity
  # noSpawn: true
  # parent: BaseCaptainObjective
  # id: NukeDiskStealObjective
  # components:
  # - type: Objective
    # # high difficulty since the hardest item both to steal, and to not get caught down the road,
    # # since anyone with a pinpointer can track you down and kill you
    # # it's close to being a stealth loneop
    # difficulty: 4
  # - type: NotCommandRequirement
  # - type: StealCondition
    # stealGroup: NukeDisk
    # owner: objective-condition-steal-station
# Corvax-MRP-End
```